### PR TITLE
Pantherlog aws cloudtrail insights digest

### PR DESCRIPTION
**Pull request by alxarch**
## Background

Port AWS.CloudTrailInsight and AWS.CloudTrailDigest to pantherlog format

## Changes

- Port parser for AWS.CloudTrailInsight 
- Port parser for AWS.CloudTrailDigest
- Port tests to YAML

## Testing

- mage test:go

### CODE REVIEW

**Review by giorgosp — APPROVED** [review]

File: internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go [hunk 1]
```diff
@@ -27,90 +27,74 @@ import (
...
+	Statistics *InsightStatistics `json:"statistics" description:"A container for data about the typical average rate of calls to the subject API by an account, the rate of calls that triggered the Insights event, and the duration, in minutes, of the Insights event."`
 }
 
 // nolint:lll
 type InsightStatistics struct {
-	Baseline        *InsightAverage `json:"baseline,omitempty" description:"Shows the typical average rate of calls to the subject API by an account within a specific AWS Region."`
-	Insight         *InsightAverage `json:"insight,omitempty" description:"Shows the unusual rate of calls to the subject API that triggers the logging of an Insights event."`
-	InsightDuration *float32        `json:"insightDuration,omitempty" description:"The duration, in minutes, of an Insights event (the time period from the start to the end of unusual activity on the subject API). insightDuration only occurs in end Insights events."`
+	Baseline        *InsightAverage    `json:"baseline" description:"Shows the typical average rate of calls to the subject API by an account within a specific AWS Region."`
```
Comment (kostaspap) [inline internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go:63]: Why remove the `omitempty` here? How does jsoniter behave if `InsightAverage` is nil? 
Same for `Statistics`, `Insight`

Comment (alxarch) [reply internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go:63]: All fields are omitempty when using `pantherlog`

**Review by kostaspap — APPROVED** [review]

File: internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go (hunk 1 above)
Comment (kostaspap) [reply internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go:63]: Spoke offline - @alxarch verified that the struct will be omitted if it is empty without the need to add an explicit `omitempty` tag.

Comment (alxarch) [reply internal/log_analysis/log_processor/parsers/awslogs/cloudtrail_insights.go:63]: `omitempty` omits `nil` values for maps, slices, pointers.

**Pull request merged**